FL_Studio/FLStudio.download.recipe

```diff
@@ -18,6 +18,13 @@
         <dict>
             <key>Arguments</key>
             <dict>
+            <key>request_headers</key>
+                <dict>
+                    <key>user-agent</key>
+                    <string>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.1 Safari/605.1.15</string>
```

Comment:
I'm open to this being baked in, but I've tended to put this in the input vars so if there's something that causes this to show different information based on the architecture or macOS version that admins can change this themselves